content/zh/docs/contribute/style/diagram-guide.md

```diff
@@ -0,0 +1,1262 @@
+
+
+{{< /mermaid >}}
+
+<!--
+Figure 4. Mermaid+SVG method steps.
+-->
+图 4. Mermaid+SVG 方法的步骤。
+
+<!--
+The following lists the steps you should follow for adding a diagram using the Mermaid+SVG method:
+
+1. Create your diagram using the live editor. 
+2. Store the diagram URL somewhere for later access.
+3. Generate an `.svg` image file for the diagram and download it to the appropriate `images/` folder. 
+4. Use the `{{</* figure */>}}` shortcode to reference the diagram in the `.md` file. 
+5. Add a caption using the `{{</* figure */>}}` shortcode's `caption` parameter.
+-->
+使用 Mermaid+SVG 方法来添加图表时你要遵从的步骤：
+
+1. 使用在线编辑器创建你的图表
+1. 将图表的 URL 保存到某处以便以后访问
+1. 为你的图表生成 `.svg` 文件，并将其下载到合适的 `images/` 目录下
+1. 使用 `{{</* figure */>}}` 短代码在 `.md` 文件中引用该图表
+1. 使用 `{{</* figure */>}}` 短代码的 `caption` 参数为图表设置标题
+
+<!--
+For example, use the live editor to create a diagram called `boxnet`.
+Store the diagram URL somewhere for later access. Generate and download a
+`boxnet.svg` file to the appropriate `../images/` folder. 
+-->
+例如，使用在线编辑器创建一个名为 `boxnet` 的图表。
+将图表的 URL 保存到别处以便以后访问。生成 `boxnet.svg` 文件并将其下载到合适的
+`../images/` 目录下。
+
+<!--
+Use the `{{</* figure */>}}` shortcode in your PR's `.md` file to reference
+the `.svg` image file and add a caption.
+-->
+在你的 PR 中的 `.md` 文件内使用 `{{</* figure */>}}` 短代码来引用
+`.svg` 图片文件，并为之添加标题。
+
+<!--
+```json
+{{</* figure src="/static/images/boxnet.svg" alt="Boxnet figure" class="diagram-large" caption="Figure 14. Boxnet caption" */>}}
+```
+-->
+```json
+{{</* figure src="/static/images/boxnet.svg" alt="Boxnet 示意图" class="diagram-large" caption="图 14. Boxnet 标题" */>}}
+```
+
+<!--
+For more details on diagram captions, see [How to use captions](#how-to-use-captions).
+-->
+关于图表标题的细节，可参阅[如何使用标题](#how-to-use-captions)。
+
+{{< note >}}
+<!--
+The `{{</* figure */>}}` shortcode is the preferred method for adding `.svg` image files
+to your documentation. You can also use the standard markdown image syntax like so:
+`![my boxnet diagram](static/images/boxnet.svg)`.
+And you will need to add a caption below the diagram. 
+-->
+使用 `{{</* figure */>}}` 短代码是向你的文档中添加 `.svg` 图片文件的优选方法。
+你也可以使用标准的 markdown 图片语法，即
+`![my boxnet diagram](static/images/boxnet.svg)`。
+如果是后面这种，则需要在图表下面为其添加标题。
+{{< /note >}}
+
+<!--
+You should add the live editor URL as a comment block in the `.svg` image file using a text editor.
+For example, you would include the following at the beginning of the `.svg` image file:
+-->
+你应该使用文本编辑器以注释块的形式在 `.svg` 图片文件中添加在线编辑器的 URL。
+例如，你应该在 `.svg` 图片文件的开头部分包含下面的内容：
+
+```
+<!-- To view or edit the mermaid code, use the following URL: -->
+<!-- https://mermaid-js.github.io/mermaid-live-editor/edit/#eyJjb ... <remainder of the URL> -->
```

Comment:
Possibly, we could mention that localized SVG files either don't need this comment, or that they should use a custom URL _for the localized diagram_.